### DISCUSSION TITLE
xPore m6ACE-seq validation set

### DISCUSSION THREAD

**Discussion by JannesSP** [GoekeLab/xpore#112] [Q&A] [answered]
Dear Göke,

in your paper "Identification of differential RNA modifications from
nanopore direct RNA sequencing with xPore" you mention a validation
set of m6ACE-seq data to find the m6A modification.
Do you have a list with the positions of validated m6A sites per read
or reference for the HEK293T WT replicates somewhere?

With kind regards
Jannes Spangenberg

**Comment by ploy-np** [comment, marked as answer]
Hi @JannesSP, 
I just uploaded in our zenodo. You can download the m6ACE-Seq dataset directly from this link here: https://zenodo.org/record/5707193/files/m6ACE-Seq.csv?download=1.
